driver-core/src/test/java/com/datastax/driver/core/ProtocolV1Test.java

```diff
@@ -24,7 +25,7 @@
 /**
  * Tests targeting protocol v1 specifically.
  */
-@CCMConfig(version = "1.2.19")
+@CassandraVersion(major = 1.2, exactRequired = true)
```

Comment:
Effectively yes.   It skips any cassandra versions that doesn't match the criteria exactly.  i.e. if `major` is 1.2, only C* versions matching 1.2.X will qualify.   If `major` is `1.2`, minor is `19`, only 1.2.19 will qualify.   I agree it's a bit crude.  @adutra has a good solution for this that allows a version range specification that I think will be much better/flexible that we can use in the future.